src/amitaRJun20/StringRotationComparision.java

```diff
@@ -0,0 +1,40 @@
+/*Coding test 10 :
+
+Print minimum rotation required to compare both strings equality.
+
+Example 1  : 
+String input1 = "abcdef";
+String input2 = "defabc";
+output : 3
+
+Example 2  : 
+String input1 = "adbcdef"; -> push right
+fadbcde
+efadbcd
+defadbc
+String input2 = "defadbc";
+output : 3*/
+package amitaRJun20;
+
+public class StringRotationComparision {
+	
+	void findMnimumRotation(String input1,String input2) {
+		int count = 0;
+		if(input1.length()!=input2.length()){
+			System.out.println("Strings are not same, lengths are different");
+		}else {	
+		while(!input1.equals(input2)) {
```

Comment:
it could be infinite loop if both string could match at all. you should add one more condition in same code, 

while(!input1.equals(input2) || count <= input1.length)